changelog/v1.19.0-beta2/issue_10456.yaml

```diff
@@ -0,0 +1,6 @@
+changelog:
+  - type: FIX
+    issueLink: https://github.com/solo-io/gloo/issues/10456
+    resolvesIssue: true
```

Comment:
Is this true? We need to update the solo-projects dep to run this in CI as well before it is considered complete imo

Reply:
^ issue is for gloo. IMHO solo-projects should have a separate issue to bump the gloo dep. I created https://github.com/solo-io/solo-projects/issues/7404 for this purpose.